thorlcr/activities/choosesets/thchoosesetsslave.cpp

```diff
@@ -250,6 +250,7 @@ class InputCounter : public CSimpleInterface, implements IThorDataLink
     virtual CActivityBase *queryFromActivity();
     virtual void dataLinkSerialize(MemoryBuffer &mb);
     unsigned __int64 queryTotalCycles() const;
+    virtual void debugRequest(MemoryBuffer &msg);
```

Comment:
Consistency.
All are virtuals and declared virtuals here, apart from queryTotalCycles(),.